lab_3/main.py

```diff
@@ -140,16 +140,12 @@ def predict_next_sentence(self, prefix: tuple) -> list:
 
 
 def encode(storage_instance, corpus) -> list:
-    corp_1 = []
-    id_sent = []
+    encode = []
     for sentence in corpus:
-        for word in sentence:
-            number = storage_instance.get(word)
-            id_sent.append(number)
-        corp_1.append(id_sent)
-        id_sent = []
-        continue
-    return corp_1
+        for key,value in storage_instance.items():
```

Comment:
@NatalieAntoshka for some reason all of you ignore provided scenario of using your lab. We are doing this lab to be able to generate some sentences/text and to be able to use it not just to enable provided unit-tests. On a given scenario this still does not work. Correct implementation looks like this:
```python
def encode(storage_instance, corpus) -> list:
    encode = []
    for sentence in corpus:
        enc_sent = []
        for word in sentence:
            enc_sent.append(storage_instance.get_id_of(word))
        encode.append(enc_sent)
    return encode
```